### ISSUE TITLE
Drum icon

### ISSUE THREAD

**Issue by Nihvel** [Templarian/MaterialDesign#4334]
I have:

- [x] [Searched all issues](https://github.com/Templarian/MaterialDesign/issues) to make sure there isn't a request for this icon.
- [x] [Searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't exist.
- [x] Only requested a single icon (or a few near-identical ones) in this issue. 

## Usage

I play the drum, I've got a few of them. I'd like to ask you for a icon for an acoustic drum and, if it is possible, one for the electronic drum.

## Examples

This might be for the acoustic drum, but this one is copyright protected:
https://iconscout.com/icon/drum-kit-3

I found a lot of icons here, might give inspiration:
https://www.flaticon.com/search?search-type=icons&word=drum

I can not find icons for the electronic drums, I fear they do not exist. If it is possible for you to make one it will be nice otherwise I'm sure the acoustic one will make me forget about the other :D

**Comment by MrGrigri** [comment]
Are you looking for a full drum set or just one like a snare?

**Comment by Nihvel** [comment]
Yes a full drumset would be best, similar to the one in the link above 'drum-kit-3'

**Comment by goyney** [comment]
Was playing with this a little earlier.

Too complex, breaks too many rules:
![image](https://user-images.githubusercontent.com/1015040/63307255-ccabe100-c2bb-11e9-994d-09faa18377f9.png)

Better:
![image](https://user-images.githubusercontent.com/1015040/63307240-bbfb6b00-c2bb-11e9-9021-c25739392a76.png)

Still needs some work, though.

**Comment by JapanYoshi** [comment]
I'm curious if Rock Band style icons would work. Examples:

Rock Band 4:
![](https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=2ahUKEwim4bvmn5DkAhXP7HMBHVJICIEQjRx6BAgBEAQ&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FFile%3AGH_Drum_Icon.svg&psig=AOvVaw0iL3mQxgpRsmbRo4ZinUNl&ust=1566349183975643)

Not sure where this is from but it's interesting:
![](https://p7.hiclipart.com/preview/115/800/950/rock-band-4-the-beatles-rock-band-computer-icons-musical-ensemble-drum-stick.jpg)

**Comment by JapanYoshi** [comment]
![drum_maybe](https://user-images.githubusercontent.com/8707990/63309948-fdcbf600-c33b-11e9-946f-09eb5bff43ff.png)
I gave it a crack, but it *is* pretty hard.

**Comment by Nihvel** [comment]
Hey, thank you all guys for joining this request! 

@goyney I like the first icon more, the one below doesn't look like a drum. But them both do not really convince me :/ Maybe because you're trying to remain on MaterialDesign style (which makes sense) and others icons I saw look more like a drumset (see examples in the links).
But honestly I really appreciate your work and the time that it took you! There's still a little space for improvements and it's not that far from the final result! (:

Suggestion 1:
You could try making the transparent hole littler and not centered. 
See here how:
https://sc1.musik-produktiv.com/pic-010068548xxl/pearl-export-20-smokey-chrome-complete-drumset.jpg
That hole can be indeed a real hole. It's where the bass drum is microphoned. 

Suggestion 2: 
Some cymbals are not at the same height; Can you try lower the left stand a bit and make it like this?:
https://s3.amazonaws.com/images.static.steveweissmusic.com/products/images/uploads/1126725_14611_popup.jpg
Thanks, thanks, thanks!

@JapanYoshi I saw it but I don't like it mate, I like tho people contributing to this and I wanna thank you for your time too!

**Comment by Nihvel** [comment]
Hey, are there updates? Tx

**Comment by Nihvel** [comment]
Hello, news?

**Comment by Nihvel** [comment]
@goyney Hi, is this project going to be followed? I can not say that a icon is urgent but you were so close the first day in showing some progress that i find it strange it's abandoned now. Thanks

**Comment by goyney** [comment]
@Nihvel This is an open-source project and we're all donating our time. As long as the issue remains open, it'll be addressed at some point.

**Comment by Outliver** [comment]
I'll opt for the single snare.

We got `guitar-acoustic` for _chordophones_, `trumpet` for _aerophones_ and you can use `bell` for _idiophones_ and `current-ac` for _electrophones_. The only thing missing is something for _membranophones_.

So, anything like a snare drum or a tom would be nice (maybe with sicks).
Thanks everyone! :)

**Comment by tgrelka** [comment]
![drum-v1](https://user-images.githubusercontent.com/22134144/79694857-71c7f080-8273-11ea-99ad-63b9b4c6e678.png)
Since I'm in need of a drum icon as well, I gave it a shot - this icon does view well at 24px, but at larger sizes becomes quite dominant. So potentially, I might make the vertical stripes thinner.
Also, I'm not sure about the sticks and might need some help with them! I'll provide SVG/Affinity Designer files on request.

**Comment by tgrelka** [comment]
![drum-v2](https://user-images.githubusercontent.com/22134144/79695018-8953a900-8274-11ea-8ab5-f21e618f0e1a.png)
I also tried a top-down variant. Not sure which one I like more. Top down seems easier to make, but a view from the side might be more recognizable.

**Comment by Outliver** [comment]
@tgrelka uuh, that's amazing! I agree with both: The top-down version is probably harder to identify by non-drummers and the other one might need thinner stripes. While I (as a musician) clearly recognize the top-down version as being a drum, in the side-view, the sticks are really the only thing giving it away. So, I think, I'd prefer the side-view if it's gonna be a little more refined. Anyway, thanks a lot! Really much appreciated! :)

**Comment by tgrelka** [comment]
Yeah, the reason the stripes are that wide is because at very small scales (which MDIs are required to scale to) it is no longer recognizable with smaller stripes. I'm still tweaking around and testing.

**Comment by tgrelka** [comment]
[Drum.zip](https://github.com/Templarian/MaterialDesign/files/4499987/Drum.zip)
For anyone wanting to help, here's a .zip with both .svgs and .afdesign files for the icons! Any help is welcome.

**Comment by Nihvel** [comment]
Thanks everybody, I appreciate it a lot that you're working on this. I have no skills with image editing but I was in need of a drum svg and I hacked my own some months ago.

I still prefer the whole drumset, and since I now have it I'm not anymore interested at a Material Design version. But please feel free to continue this 'issue'. MD needs a drum icon.

[drumset.zip](https://github.com/Templarian/MaterialDesign/files/4529952/drumset.zip)

**Comment by dixy52-beep** [comment]
I tried to make it. 

[drums.zip](https://github.com/Templarian/MaterialDesign/files/5283089/drums.zip)

**Comment by goyney** [comment]
@dixy52-beep Can you create a preview and add it for us to see please? http://dev.materialdesignicons.com/contribute/github

**Comment by dixy52-beep** [comment]
![download (4)](https://user-images.githubusercontent.com/71886581/94277432-d8031e80-ff49-11ea-817b-88dddf3be5c2.png)
Here you are!

**Comment by visualpharm** [comment]
You can take our https://icons8.com/icon/V-ajeGKOqst0/marching-tenor-drums if you'd like.

https://img.icons8.com/material-outlined/96/000000/Marching-Tenor-Drums.png

**Comment by JapanYoshi** [comment]
@dixy52-beep - Not liking it. It's mushy at 1x and the lines aren't 2dp wide.
@visualpharm It looks like a muffin or chef's hat with . . . wings? vibrations? two different shadows? I don't know, It confuses me.

**Comment by suterma** [comment]
Chiming in late. I need to say that all the earlier proposals actually depict a drum KIT, not a drum. I think there is room for two icons, a "instrument-drum-kit" and a "instrument-drum". I feel this thread is about the drum. Thus my suggestion is to keep it really simple: Show a circle, with two sticks, one on each side. This would the the "top view", recognisable for most drummers, which I tend to think would serve the need for such an icon.

**Comment by Outliver** [comment]
How about bongos, a djembe or a conga? Should be much more recognizable.

**Comment by Nihvel** [comment]
I like it a lot, the top-down version made by @tgrelka with drumsticks at "90 degree" (okay, less than 90). But also the simplicity of the models provided by @suterma.
I would think that a good icon is the combination of these two. 

This is not what I had in mind when I created this issue, because I needed a drumkit. But I believe that the association snare+drumsticks give already a good idea.

Sure that a drumkit is imho also nice to have, whether it is a jazz or metal drumkit (with more, or less, bum bum tss), Just like bongos, djembre and conga.

**Comment by foobarth** [comment]
Bump. Is there any progress regarding a drum icon? I am by no means an artist, so i wait patiently for quite some time now :-) IMHO most above suggestions look great, so why not pick one and merge?
